packages/models-library/src/models_library/projects.py

```diff
@@ -110,17 +122,15 @@ class Project(BaseProjectModel):
         ..., description="user email", alias="prjOwner"
     )
 
-    # Timestamps   TODO: should we use datetime??
-    creation_date: str = Field(
+    # Timestamps
+    creation_date: DateTimeStr = Field(
         ...,
-        regex=DATE_RE,
         description="project creation date",
         examples=["2018-07-01T11:13:43Z"],
```

Comment:
this very much looks like the ISO timestamp.

Reply:
@sanderegg it was already like that. I did not dare changing it. But I will try in next PR and see what happens :-)